source/btle/btle.cpp

```diff
@@ -191,8 +191,11 @@ static void btle_handler(ble_evt_t *p_ble_evt)
 
             // Close all pending discoveries for this connection
             nRF5xGattClient& gattClient = ble.getGattClient();
+#if  defined(TARGET_MCU_NRF51_16K_S110) || defined(TARGET_MCU_NRF51_32K_S110)
```

Comment:
Hi @ddavidebor 
I also encountered this issue, so thanks for reporting it. would you like be so kind and update the pull request as requested (which makes it easier for the maintainer to include the fix in the tree)? otherwise I'll open a second one...
```
#if !defined(TARGET_MCU_NRF51_16K_S110) && !defined(TARGET_MCU_NRF51_32K_S110)
            // Close all pending discoveries for this connection
            nRF5xGattClient& gattClient = ble.getGattClient();
            gattClient.characteristicDescriptorDiscoverer().terminate(handle, BLE_ERROR_INVALID_STATE);
            gattClient.discovery().terminate(handle);
#endif
```